extra-web/palemoon/autobuild/defines

```diff
@@ -1,7 +1,7 @@
 PKGNAME=palemoon
 PKGSEC=web
 PKGDEP="dbus-glib desktop-file-utils gst-plugins-base-1-0 gtk-2 \
```

Comment:
Are you sure that `gtk-2` is still needed?

Reply:
Yes or the build will fail. I have tested it before filling PR.

Reply:
Got it.